rollup.config.js

```diff
@@ -23,6 +32,7 @@ export default {
       preferBuiltins: false,
     }),
     commonjs(),
+    nodePolyfills(),
```

Comment:
Does this increase the bundle size a lot? I wonder if this is just something we should have in the commonJS export?

Reply:
IIRC this was meant to fix an issue with one of the connectors trying to load a Node module, but I can’t reproduce the issue so I removed it for now, but it didn’t seem to impact the size much.